otx/algorithms/segmentation/configs/ocr_lite_hrnet_18/model.py

```diff
@@ -22,7 +22,6 @@
 
 model = dict(
     type="ClassIncrSegmentor",
-    is_task_adapt=True,
```

Comment:
Doesn't semi-sl support task adaptation?
As far as I know, task adapt is just for sampler setting